src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj

```diff
@@ -32,17 +32,18 @@
     </XsdSchemaFile>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="InstrumentationConfigurationSources.xsd" />
-    <EmbeddedResource Include="EngineConfiguration.xsd" />
+    <EmbeddedResource Include="Schema\InstrumentationConfigurationSources.xsd" />
+    <EmbeddedResource Include="Schema\EngineConfiguration.xsd" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Diagnostics.NETCore.Client" Version="0.2.61701" />
     <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="$(MicroBuildCoreVersion)" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20104.2" />
```

Comment:
>CommandLine [](start = 38, length = 11)

How much size overhead does this add to our distribution? This makes a lot of sense for command line tools (i.e. dotnet tools all use this library) but this seems unnecessary for something that isn't spawned directly by the user.

Reply:
The difference in size for the extracted Release builds is about 150 KB (from 4.1 MB to 4.25 MB). I think the benefits of parameter validation and help text are worth the small size growth

---
In reply to: [394139546](https://github.com/microsoft/CLRInstrumentationEngine/pull/234#discussion_r394139546) [](ancestors = 394139546)